### PR TITLE
backup and restore procedure #2851

### PR DESCRIPTION
**Pull request by ylorph**

### CODE REVIEW

File: docs/operations/database-backup.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ There are 2 main ways to perform backups:
 
 ### Disk Snapshotting 
 
-If you the infrastructure is virtualized, disk _snapshots_ is an option and the easiest to perform backup and restore operations.
+If your infrastructure is virtualized, disk _snapshots_ is an option and the easiest to perform backup and restore operations.
```
Comment (ahjohannessen) [inline docs/operations/database-backup.md:11]: You could mention ZFS (and perhaps BTRFS) as the superior way to keep your data safe and take snapshots that you can ship to different servers and media.

**Comment by ylorph** [comment]
resolves EventStore/documentation#230

File: docs/operations/database-backup.md (hunk 1 above)
Comment (condron) [reply docs/operations/database-backup.md:11]: We should add this to a dedicated section on ZFS.

**Review by hayley-jean — DISMISSED** [review]

**Review by condron — APPROVED** [review]
looks good

**Review by hayley-jean — APPROVED** [review]

**Pull request merged**